client/app/core/authorization.config.js

```diff
@@ -80,13 +86,16 @@ export function authInit($rootScope, $state, Session, $sessionStorage, logger, L
       .then(Language.onReload)
       .then(ServerInfo.set)
       .then(ProductInfo.set)
-      .then(rbacReloadOrLogin(toState, toParams))
+      .then(function() {
```

Comment:
Since this functionality is resolving the promise, I can't move this functionality out to another function since that would be out of scope.